packages/sdks/python/tester-local/tests/test_get_doc_query_status.py

```
@@ -0,0 +1,25 @@
+
+load_dotenv()
+
+api_key = os.environ.get("API_KEY")
+patient_id = os.environ.get("PATIENT_ID")
+facility_id = os.environ.get("FACILITY_ID")
+base_url = os.environ.get("BASE_URL")
+
+
+def test_start_doc_query() -> None:
+    metriport = Metriport(api_key=api_key, base_url=base_url)
+    response = metriport.medical.document.start_query(patient_id=patient_id, facility_id=facility_id)
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)